service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -572,6 +614,13 @@ public String getRelevantProceedings() {
         return Collections.unmodifiableList(othersList);
     }
 
+    @JsonIgnore
+    public List<Element<Other>> getAllActiveOthers() {
+        return getAllOthers().stream()
+            .filter(other -> YES.getValue().equals(other.getValue().getActiveParty()))
```

Comment:
Do we want to use
```
!NO.getValue().equals(other.getValue().getActiveParty()
```
and in that case we can consider those that have not been given a value to be active as well so that all current data is considered active